dawn/src/dawn/Optimizer/Lowering.cpp

```diff
@@ -652,9 +633,11 @@ bool OptimizerContext::fillIIRFromSIR(
   PassTemporaryType::fixTemporariesSpanningMultipleStencils(
       stencilInstantiation.get(), stencilInstantiation->getIIR()->getChildren());
 
-  if(getOptions().ReportAccesses) {
-    stencilInstantiation->reportAccesses(dawn::log::info.stream());
-  }
+  // Options are not available here
+  // TODO Move this elsewhere or delete
+  // if(getOptions().ReportAccesses) {
+  //   stencilInstantiation->reportAccesses(dawn::log::info.stream());
+  // }
```

Comment:
Should this be solved within this PR?

Reply:
Btw there is an `options` object available, should be the one you need, right?

Reply:
Good catch!